_includes/build/best_practices_429.html

```diff
@@ -0,0 +1,15 @@
+
+<div>
+    <p>Occasionally, when interacting with BCDA, you might encounter responses with a status code of <code>429 Too Many Requests</code>.</p>
+  
+    <p>BCDA assigns 429 response codes based on two independent criteria:</p>
+  
+    <ol>
+      <li>Making too many HTTP requests in a period of time.</li>
+      <li>Attempting to create duplicate jobs.</li>
+    </ol>
+  
+    <p>Regardless of the reason, BCDA will include a <strong>Retry-After</strong> header in its response. It's considered best practice to wait until the period specified in this header has elapsed before making further requests. This ensures your client can adapt without manual intervention, even if the rate-limiting parameters change.</p>
+  
+    <p>Concerning duplicate jobs, BCDA defines them as attempts to recreate a job already marked as "In-Progress". For reference, you can view both existing and past jobs by accessing the job history endpoint at <code>/jobs</code>.</p>
+</div>
```

Comment:
Pasting Jennifer's suggestions in:

> A status code of 429 indicates “Too Many Requests.” This can occur due to two reasons:
> 
> 1. ﻿﻿﻿Making too many HTTP requests within a period of time
> 2. Trying to recreate jobs already marked as "In-Progress.” For reference, you can view both existing and past jobs using the /jobs endpoint. See [Request Jobs History](https://bcda.cms.gov/build.html#jobs-history) for more details.
> 
> Regardless of the reason, you will see a Retry-After response in the header. Wait until the period of time specified in the header has passed before making any more requests. This makes sure your client can adapt without manual intervention, even if the rate-limiting parameters change.